src/pages/components/link/usage.mdx

```diff
@@ -81,86 +88,167 @@ be used for navigational actions.
 <Row>
 <Column colLg={8}>
 
-![link anatomy image](images/01_link_anatomy_608.png)
+![Link anatomy image](images/link-usage-anatomy.png)
 
 </Column>
 </Row>
 
-1. **Label:** Communicates what is being linked to.
+1. **Link text:** Communicates what is being linked to
+2. **Icon (optional)** Indicates the destination or type of content being
+   referenced
 
 ### Sizing
 
 There are three size variants for the link component: small, medium, and large.
-The link size should match the type size of the text it is inline with. When
-using links apart from other content, the link size should match the default
-body copy size of the page.
+Inline link sizes should match the type size of the text it is inline with.
+Standalone link sizes should match the default body copy size of the page.
+
+| Size   | Height (px / rem) |
+| ------ | ----------------- |
+| Small  | 16 / 1            |
+| Medium | 18 / 1.125        |
+| Large  | 22 / 1.375        |
+
+<Row>
+<Column colLg={8}>
+
+![Sizing applies to both standalone and inline links](images/link-usage-size.png)
+
+</Column>
+</Row>
+
+<Caption>Sizing applies to both standalone and inline links</Caption>
+
+### Placement
+
+Inline links usually sit within a sentence or paragraph, while standalone links
+can sit on their own, separated from other text. Consider reducing the number of
+inline links in a single sentence to simplify the message.
+
+<Row>
+<Column colLg={8}>
+
+![Example of inline link in a notification](images/link-usage-placement-1.png)
+
+</Column>
+</Row>
+
+<Caption>Example of inline link in a notification</Caption>
 
 <Row>
 <Column colLg={8}>
 
-![Examples of link sizes](images/03_link_size_608.png)
+![Example of standalone link in a card](images/link-usage-placement-2.png)
 
 </Column>
 </Row>
 
+<Caption>Example of standalone link in a card</Caption>
+
 ## Content
 
-We recommend links be three words or fewer. Because links take users to a new
-location, it is important that their labels accurately reflect the content users
-will find at the link destination. Use meaningful labels for links and avoid
-terms like "click here" or the web address itself.
+#### Link text
+
+- Use distinctive and meaningful link text.
+- Ensure that links accurately reflect the content users will find at the link
+  destination.
+- If your design uses generic link names such as “read more,” consider making
+  them unique. See
```

Comment:
```suggestion
  them unique. See the
```

Reply:
I would link to the anchor link section "Ensure link context" that this text is referencing.